src/lib/components/Appbar.svelte

```diff
@@ -1,18 +1,25 @@
 <script>
+	import { getSidebarItems } from '$lib/utils';
 	import { AppBar, LightSwitch } from '@skeletonlabs/skeleton';
 	import {
 		IconBrandDiscordFilled,
 		IconBrandGithubFilled,
 		IconHome,
 		IconMenu2
 	} from '@tabler/icons-svelte';
+
+	const sideBarItems = getSidebarItems();
+	export let toggleSidebar = false;
 </script>
 
 <AppBar shadow="shadow-2xl" slotTrail="!space-x-2">
 	<svelte:fragment slot="lead">
 		<div class="flex items-center space-x-4">
 			<!-- Hamburger Menu -->
-			<button class="btn-icon btn-icon-sm lg:!hidden">
+			<button
```

Comment:
Use medium instead of large

![image](https://github.com/Samoa-Stack-Overflow/sso-website/assets/20700150/27698e35-f14d-437b-904a-c6bc87566d87)
![image](https://github.com/Samoa-Stack-Overflow/sso-website/assets/20700150/1b0c61ae-8282-45ad-8b59-8ad0fdd8df45)

Reply:
Done!